etc/ReplayOfflineConfiguration.py

```diff
@@ -37,6 +38,8 @@
 # 367102 - Collisions 2023 - 1200b - 0.5h long - all components IN
 setInjectRuns(tier0Config, [369998])
 
+setInjectLimit(tier0Config, 10)
```

Comment:
Could you please comment out the call? We don't want the limitation to be the default in the master branch:
```suggestion
# Use this in order to limit the number of lumisections to process
# setInjectLimit(tier0Config, 10)
```